src/pages/Quiz/QuestionWrapper/index.js

```diff
@@ -49,6 +51,7 @@ const QuestionWrapper = (props) => {
     match: { params: { testNumber = null } = {} } = {},
     selectedQuestion = null,
     testTitle = null,
+    questions = 0,
```

Comment:
why do you use `0` as default value instead of an empty array?

Reply:
fixed